sys/dev/php-fpm/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM php:8.0-fpm-alpine
```

Comment:
not a big fan of having different configurations for dev and prod. for me, they should match 1:1.